score/mw/com/impl/tracing/configuration/tracing_filter_config_test.cpp

```diff
@@ -727,5 +727,135 @@ TEST_F(TracingFilterConfigGetNumberOfTracingSlotsDeathTest, ConfigurationContain
     EXPECT_DEATH(tracing_filter_config_.GetNumberOfTracingSlots(configuration_.value()), ".*");
 }
 
+using TracingFilterConfigGetNumberOfTracingSlotsFixture = ConfigurationFixture;
```

Comment:
I see quite extensive code duplication between the tests. Did you consider refactoring parts of the setup into helper functions/fixtures that you can reuse?